### PR TITLE
Make identifier(DECTID/DA1) string non-configurable.

### PR DESCRIPTION
**Pull request by sam-mccall**
It's hard to see compelling use cases: most software will assume
vt102+terminfo anyway.

But even if this should be configurable, configuring the ident string
directly isn't the right mechanism - it should be coupled to configuring
the reported features.

### CODE REVIEW

**Review by chandlerc — APPROVED** [review]

**Comment by sam-mccall** [comment]
FYI: I can't merge these, even after approval - I think you need to merge them or grant me write access to the repo so I can do it.

**Comment by chandlerc** [comment]
Manually rebased and pushed for now in 941513e845a7a8707c1cba0234b38200acd1b60c.

**Pull request closed without merging**
